packages/lesswrong/lib/collections/petrovDayActions/collection.ts

```diff
@@ -0,0 +1,13 @@
+import schema from './schema';
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields } from '../../collectionUtils'
+
+export const PetrovDayActions: PetrovDayActionsCollection = createCollection({
+  collectionName: 'PetrovDayActions',
+  typeName: 'PetrovDayAction',
+  schema
```

Comment:
you'll need to add the default mutations.  `editCheck` shold return false.  `removeCheck` should return false.

Reply:
`newCheck` will, in the future, need to run a query against previous actions taken by this user, to see if they have a specific role that allows them to take the action that they're trying to take with the new record